src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/Microsoft.AspNetCore.Razor.OmniSharpPlugin.csproj

```diff
@@ -5,10 +5,6 @@
 
     <Description>Razor is a markup syntax for adding logic to pages. This package contains the Omnisharp Razor plugin that extracts Razor configuration information from projects.</Description>
     <EnableApiCheck>false</EnableApiCheck>
-
-    <!-- These pieces are required in order to reference OmniSharp.MSBuild -->
-    <SignAssembly>false</SignAssembly>
```

Comment:
I suppose these aren't needed anymore because the new versions are strong-named?

Reply:
Yup exactly!